requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+pytest
+https://github.com/unt-libraries/pyuntl/tarball/python2to3#egg=pyuntl
```

Comment:
I think pyuntl should go into a `requirements.txt` file, along with lxml and pypairtree.

Reply:
Also, we wouldn't want to keep the python2to3 branch in here, we'd want it to use master.

Reply:
I'm using python2to3 branch for now as the python3 version code is not yet merged into master branch of pyuntl project yet.